Add note about Hermes path on RN 0.69 (#5202)
getsentry/sentry-docs: src/platforms/react-native/manual-setup/hermes.mdx

node_modules/hermes-engine/{OS-BIN}/hermesc -O -emit-binary -output-source-map -out=main.jsbundle.hbc main.jsbundle
```

<Note>

> If you're using React-Native >= v0.69, the path of `hermesc` has changed and it is now `node_modules/react-native/sdks/hermesc/{OS-BIN}/hermesc`.

Author: ```suggestion
If you're using React Native >= v0.69, the path of `hermesc` has changed and it is now `node_modules/react-native/sdks/hermesc/{OS-BIN}/hermesc`.
```

> If you're using React Native >= v0.69, the path of `hermesc` has changed and it is now `node_modules/react-native/sdks/hermesc/{OS-BIN}/hermesc`.

Reviewer: Just a couple minor wording edits

```suggestion
If you're using React Native v0.69 or higher, the path of `hermesc` has changed to `node_modules/react-native/sdks/hermesc/{OS-BIN}/hermesc`.
```
